### ISSUE TITLE
After creating new data type from modal window unable to close alert

### ISSUE THREAD

**Issue by PeterKvayt** [umbraco/Umbraco-CMS#17171]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

14.2.0

### Bug summary

I tried to close alert by clicking on "close" icon, but nothing happen.

![image](https://github.com/user-attachments/assets/b1499c32-8e4d-4694-9b21-ad50a204a1fa)

### Steps to reproduce

1. Create new document type "A"
2. Add new property
3. In modal window "Edit property" click "Select Property Editor"
4. In modal window "Select editor" click "Decimal" -> "Create new"
5. In opened modal window for editing new editor click "Save"
6. Then alert is displayed "Created"
7. Try to close it by clicking on cross icon

### Expected result / actual result

Expected: alert closed and dissapiar
Actual: unable to interact with it. I waited long time but it does not dissapiared. Only after clicking "Submit" button in "Add property" modal window I have abillity to interact with alert.

**Comment by madsrasmussen** [comment]
duplicate of https://github.com/umbraco/Umbraco-CMS/issues/16486

**Comment by madsrasmussen** [comment]
Duplicate of https://github.com/umbraco/Umbraco-CMS/issues/16486
